lightpath/ui/gui.py

```diff
@@ -96,39 +107,28 @@ def __init__(self, controller, beamline=None,
             idx = 0
         # Move the ComboBox
         self.destination_combo.setCurrentIndex(idx)
-        # Add all of our device type options
-        max_columns = 3
-        for i, row in enumerate(np.array_split(self.shown_types,
-                                               max_columns)):
-            for j, device_type in enumerate(row):
-                # Add box to layout
-                box = QCheckBox(device_type.__name__)
-                box.setChecked(True)
-                self.device_types.layout().addWidget(box, j, i)
-                # Hook up box to hide function
-                self.device_buttons[box] = device_type
-                box.toggled.connect(self.filter)
         # Setup the UI
         self.change_path_display()
         self.resizeSlider()
         # Change the stylesheet
         if dark:
             typhos.use_stylesheet(dark=True)
 
+        self.setWindowTitle('Lightpath')
```

Comment:
Title change, yes! 😁 

```suggestion
        self.setWindowTitle(f'Lightpath - {beamline}' if beamline else "Lightpath")
```